src/components/LoanCards/KivaClassicBasicLoanCardExp.vue

```diff
@@ -0,0 +1,395 @@
+					<summary-tag
+						class="tw-absolute tw-bottom-2 tw-right-1 tw-text-primary"
+						:city="city"
+						:state="state"
+						:country-name="countryName"
+					>
+						<kv-material-icon
+							class="tw-h-2.5 tw-w-2.5 tw-mr-0.5"
+							:icon="mdiMapMarker"
+						/>
+						{{ formattedLocation }}
+					</summary-tag>
+				</div>
+			</router-link>
+		</div>
+
+		<!-- Borrower name-->
+		<kv-loading-placeholder
+			v-if="isLoading"
+			class="tw-mb-0.5" :style="{width: 75 + (Math.random() * 15) + '%', height: '1.875rem'}"
+		/>
+
+		<borrower-name
+			v-if="!isLoading"
+			class="tw-mb-1 tw-text-h3"
+			:max-length="50"
+			:name="borrowerName"
+			style="min-height: 3rem;"
+		/>
+
+		<!-- LoanUse  -->
+		<kv-loading-paragraph
+			v-if="isLoading"
+			class="tw-mb-1.5 tw-flex-grow" :style="{width: '100%', height: '5.5rem'}"
+		/>
+
+		<loan-use
+			v-if="!isLoading"
+			class="tw-mb-2.5 tw-flex-grow"
+			:loan-use-max-length="52"
+			:loan-id="`${allSharesReserved ? '' : loanId}`"
+			:use="loan.use"
+			:name="borrowerName"
+			:status="loan.status"
+			:loan-amount="loan.loanAmount"
+			:borrower-count="loan.borrowerCount"
+		/>
+
+		<!-- Matching text  -->
+		<kv-loading-placeholder
+			v-if="isLoading"
+			class="tw-mb-1" :style="{width: 75 + (Math.random() * 15) + '%', height: '1.3rem'}"
+		/>
+
+		<loan-matching-text
+			v-if="!isLoading && loan.matchingText !== '' && !isMatchAtRisk"
+			class="tw-mb-1.5"
+			:matcher-name="loan.matchingText"
+			:match-ratio="loan.matchRatio"
+			:status="loan.status"
+			:funded-amount="loan.loanFundraisingInfo.fundedAmount"
+			:reserved-amount="loan.loanFundraisingInfo.reservedAmount"
+			:loan-amount="loan.loanAmount"
+		/>
+	</div>
+</template>
+
+<script>
+import { mdiChevronRight, mdiMapMarker } from '@mdi/js';
+import gql from 'graphql-tag';
+import * as Sentry from '@sentry/vue';
+import { isMatchAtRisk } from '@/util/loanUtils';
+import LoanUse from '@/components/BorrowerProfile/LoanUse';
+import percentRaisedMixin from '@/plugins/loan/percent-raised-mixin';
+import timeLeftMixin from '@/plugins/loan/time-left-mixin';
+import BorrowerImage from '@/components/BorrowerProfile/BorrowerImage';
+import BorrowerName from '@/components/BorrowerProfile/BorrowerName';
+import KvLoadingPlaceholder from '@/components/Kv/KvLoadingPlaceholder';
+import KvLoadingParagraph from '@/components/Kv/KvLoadingParagraph';
+import LoanProgressGroup from '@/components/LoanCards/LoanProgressGroup';
+import LoanMatchingText from '@/components/LoanCards/LoanMatchingText';
+import SummaryTag from '@/components/BorrowerProfile/SummaryTag';
+import KvButton from '~/@kiva/kv-components/vue/KvButton';
+import KvMaterialIcon from '~/@kiva/kv-components/vue/KvMaterialIcon';
+
+const loanQuery = gql`query kcBasicLoanCard($basketId: String, $loanId: Int!) {
+	shop (basketId: $basketId) {
+		id
+		basket {
+			id
+			# for isInBasket
+			items {
+				values {
+					id
+				}
+			}
+		}
+	}
+	lend {
```

Comment:
as you remove additional view components please also remove any data fields that aren't used or passed down